docs/docs/reference/new-types/match-types.md

```diff
@@ -103,7 +103,7 @@ The first rule is a structural comparison between two match types:
 ```
   S <: T,  m <= n,  Ci <: Di for i in 1..n
```

Comment:
`S <: T,  m <= n,  Ci <: Di for i in 1..m` (btw, this should be from `1..m` no `1..n`). 

Can you also fix it?

Reply:
I think it would make more sense that `m` belongs to `S` and `n` to `T`, I guess that is also the reason I (and others) did not spot the mistake immediately. I'll change it

Reply:
Sounds good